ui/utilities/media-objects/flavors/base/_index.scss

```diff
@@ -62,7 +62,7 @@
     }
 
     &.slds-media--small .slds-media__figure {
-      margin-left: $spacing-xx-small;
+      margin-#{$left}: $spacing-xx-small;
```

Comment:
That's how I did it when I did `rtl` for kanban (many, many moons ago). I built everything off the `[dir=...`